as.c

```diff
@@ -84,6 +97,13 @@ static void assemble_line(vm_env *env, char *line)
     const struct instruction *inst = find_inst(mnemonic);
 
     if (!inst) {
+        /* Check label */
+        if (strlen(mnemonic) > 1 &&
+                mnemonic[strlen(mnemonic) - 1] == ':') {
```

Comment:
`strlen(mnemonic)` is not always changed here. You don't have to invoke it several times.